tests/core/utils/test_kpts.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+
+import math
+
+from quacc.utils.kpts import bandgap_to_kspacing
+
+
+def test_bandgap_to_kspacing():
+    out = bandgap_to_kspacing(0.60)
+    assert math.isclose(out, 0.2004, rel_tol=1e-6)

```

Comment:
Small tip:

It is generally recommended to use pytest's `approx` tool here.

e.g. 

```
assert out == pytest.approx(0.2004) # with optional tolerance if you wish
```